lib/runners/processor/checkstyle.rb

```diff
@@ -147,12 +147,14 @@ def build_links(rule_id)
     end
 
     def config_file
-      file = config_linter[:config] || "google"
+      file = config_linter[:config] || "google" # Change "sider" when recommended ruleset is released.
```

Comment:
[suggestion] How about adding `TODO`?

```suggestion
      file = config_linter[:config] || "google" # TODO: Change "sider" when recommended ruleset is released.
```

Reply:
added "TODO".